util/http/retry.go

```diff
@@ -27,13 +28,101 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// defaults based on a target total retry time of at least 5s. 30*((2^8)-1)>5000
 const (
-	DefaultMaxRetries        = 8
-	DefaultMinWaitMsec int64 = 30
-	DefaultMaxWaitMsec int64 = 300000
+	// DefaultDialTimeoutMsec is the default number of milliseconds before timeout while connecting to a remote endpoint. See `TimeoutConfig.DialTimeout`.
+	DefaultDialTimeoutMsec = 3000
+	// DefaultResponseHeaderTimeoutMsec is the default number of milliseconds before timeout while waiting for response header from a remote endpoint. See `TimeoutConfig.ResponseHeaderTimeout`.
+	DefaultResponseHeaderTimeoutMsec = 3000
```

Comment:
These 2 are changed from previous 1 sec right?

Reply:
Yeah - we are bumping the granular timeouts from 1 - 3 seconds.

Reply:
Yeah. Would it be better to have one commit that sets them to 1s and a second commit that increases them to 3s?

Reply:
Both work for me :) it's not a big change (1 -> 3) after all.